.jenkinsci/debug-build.groovy

```diff
@@ -36,6 +43,10 @@ def doDebugBuild(coverageEnabled=false) {
     + " -v /var/jenkins/ccache:${CCACHE_DIR}") {
 
     def scmVars = checkout scm
+    def cmakeOptions = ""
```

Comment:
Maybe build type?

Reply:
Entering this function we already know which build type is selected for the job. So, no, no need to define build type here